apps/cowswap-frontend/src/modules/permit/state/permitCacheAtom.ts

```diff
@@ -14,7 +14,7 @@ import {
  * Should never change once it has been created.
  * Used exclusively for quote requests
  */
-export const staticPermitCacheAtom = atomWithStorage<PermitCache>('staticPermitCache:v2', {})
+export const staticPermitCacheAtom = atomWithStorage<PermitCache>('staticPermitCache:v3', {})
```

Comment:
Bumping the cache atoms to force recheck local cache and get new supported tokens 🤞

Reply:
We would need to add a script to clear previous versions from localStorage

Reply:
Do we have to?

Reply:
Otherwise localStorage will be polluted and some a day we can hit the storage size limit

Reply:
Not need to work on this right now, just fyi